src/modules/auth/containers/ledger-connect.js

```diff
@@ -6,7 +6,7 @@ import LedgerConnect from 'modules/auth/components/ledger-connect/ledger-connect
 import loginWithLedger from 'modules/auth/actions/login-with-ledger'
 
 const mapStateToProps = state => ({
-  networkId: augur.rpc.getNetworkID(),
+  networkId: parseInt(augur.rpc.getNetworkID(), 10),
```

Comment:
Ethereumjs-ledger was complaining about receiving a string rather than a number.  IMO it’s an integration detail, so probably don’t need to change everywhere.